### DISCUSSION TITLE
希望Bilibili Evolved可以增加获取B站直播间在线直播的直播流地址链接的功能。

### DISCUSSION THREAD

**Discussion by Yukihana-Lamy** [the1812/Bilibili-Evolved#3385] [功能建议 / Ideas]
RT，希望Bilibili Evolved可以增加获取B站直播间在线直播的直播流地址链接的功能。方便Bilibili Evolved的用户们复制和粘贴直播流地址链接，使用播放器在线观看Bilibili的在线直播。最好可以添加直播录屏功能。

**Comment by the1812** [comment]
重复 #633
